examples/nlp/language_modeling/conf/megatron_gpt_config.yaml

```diff
@@ -1,5 +1,6 @@
 name: megatron_gpt
 restore_from_path: null # used when starting from a .nemo file
+gradient_as_bucket_view: True
```

Comment:
Why is this at the highest level of the config? Should it be in `model` ?

Reply:
+1

Reply:
Done